demos/chips.scss

```diff
@@ -17,6 +17,10 @@
 @import "./common";
 @import "../packages/mdc-chips/mdc-chips";
 
+.demo-chip {
```

Comment:
I am wondering whether we should be building in at least the left/right margins if not also even top/bottom, since chips are intended to be in sets and I assume we have an opinion on how chips within a set are spaced apart? (Even if we make that style specific to chip-set...)

Reply:
+1, let's give each chip a baseline margin.

**However**, we should clarify with design whether the margins should also push the chips Npx away from the bounding box of their parent container (i.e., the chip set).

E.g.:

![image](https://user-images.githubusercontent.com/409245/36283426-5c71de08-1259-11e8-8e16-a6d7a6ffe1f0.png)

vs.:

![image](https://user-images.githubusercontent.com/409245/36283394-438c14a8-1259-11e8-8e61-d09cfbc30d78.png)

(the grey borders are only to show the boundaries of the `.mdc-chip-set` element)

It makes a difference when there's a heading, or input, or some other element nearby that needs to be horizontally aligned.

I also strongly suggest that we change `.mdc-chip-set` to use `display: flex; flex-wrap: wrap;` to collapse the whitespace in the markup, because it will give us much better control over spacing (so we can match the spec).

Reply:
Design confirmed that the margins should also push the chips away from the chip-set border.
I've hard-coded a baseline margin on the chips for now and filed #2265 for a separate PR.